src/main/kotlin/com/example/delivery/repository/ProductRepository.kt

```diff
@@ -9,10 +10,9 @@ import org.springframework.stereotype.Repository
 interface ProductRepository {
     fun findById(id: String): MongoProduct?
     fun findAll(): List<MongoProduct>
-    fun findByName(name: String): MongoProduct?
     fun save(product: MongoProduct): MongoProduct
     fun deleteById(id: String)
     fun existsById(id: String): Boolean
     fun update(id: String, update: Update): MongoProduct?
-    fun updateProductsAmount(products: Map<String, Int>)
+    fun updateProductsAmount(products: List<MongoOrder.MongoOrderItem>)
```

Comment:
```suggestion
    fun updateProductsAmount(products: List<MongoOrderItem>)
```
Maybe just like that with static import